rllib/evaluation/rollout_worker.py

```diff
@@ -334,8 +330,10 @@ def wrap(env):
             # Deepmind wrappers already handle all preprocessing
             self.preprocessing_enabled = False
 
-            if clip_rewards is None:
-                clip_rewards = True
+            # If clip_rewards not explicitly set to False, switch it
```

Comment:
clip_rewards has been moved into base-Policy's `postprocess_trajectory` method.

Reply:
Won't this break reward clipping for policies that don't call that method?

Reply:
True, if users don't use the build_policy methods, then yes. The thought was: reward clipping is actually a post-processing step.
Undid these changes again.

Reply:
Sounds good. Let's be careful about breaking changes like that.

Reply:
+1